### DISCUSSION TITLE
Please Improve Install Instructions

### DISCUSSION THREAD

**Discussion by bellemanwesley** [mitre/caldera#2245] [General]
I've installed this so many times and there's a lot of manual commands and Googling Can you add a .sh file which installs all of the packages for the correct operating system? Also the requirements.txt doesn't work because wheel is required for some of these and it does not install wheel before those packages.

**Comment by JustasZab** [comment]
Second this. Also as part of the installation you really should include instructions (or make it part of the script) to register caldera as a system service. Otherwise, if it's not a docker install and you're running it on a remote box, the server is done as soon as the ssh session is finished.

**Comment by wbooth** [comment]
Thank you both, we'll add better installation instructions (https://caldera.readthedocs.io/en/latest/Installing-CALDERA.html) and steps for both caldera and for using docker. I will leave this open until we have improved them.
